docs/src/snap/explanation/installation-methods.md

```diff
@@ -0,0 +1,67 @@
+# Choosing an installation method
+
+{{ product }} can be installed in a variety of ways, depending on your needs and
+preferences. All installation methods provide a fully functional cluster with
+upstream Kubernetes and additional built-in features such as networking, ingress
+and storage. Your choice may be influenced by the cluster size, the 
+environment you are deploying to, and the life cycle management tools that you
+prefer to use. The following sections describe the different installation
+methods:
+
+## Snap
+
+The {{ product }} snap is a self-contained, simple to install package. It can
+be installed on any Linux distribution that supports the
+[snapd] service, such as
+[Ubuntu]. If you're using a non-Linux system, we recommend creating virtual
+machines using [Multipass] or [Lxd]. [Snaps] come with the advantage of
+automatic and atomic updates making it easy for users to install, maintain and
+update their software.
+
+If you are looking for a straightforward way to get started with {{ product }},
+and you don't intend to grow your cluster beyond a single node, installing
+{{ product }} using the snap is the recommended installation method. Follow the
+[getting-started guide] to install {{ product }} using the snap.
+
+## Juju
+
+[Juju] is an open-source orchestration engine that allows you to
+deploy, configure, scale and operate {{ product }} on any infrastructure. Juju
+uses charms which are encapsulated reusable software packages to define how
+applications are deployed and related to each other. At its core, {{ product }}
+comprises two Juju charms: a control plane charm and a worker charm.
+Additionally, the Juju charm ecosystem provides further integrations, for
+example an observability stack.
+
+If you are looking to deploy {{ product }} in a public/private cloud
+environment, on metal or in VMs you can take advantage of Juju's powerful
+lifecycle management. Get started with a simple deployment of the
+{{ product }}'s charms using the [juju-cli guide] or leverage
+[Terraform's Juju provider] by following the [installing-with-terraform]
+guide.
+
+## Cluster API (CAPI)
+
+[Cluster API] is a Kubernetes sub-project that provides declarative APIs for
+creating, configuring, and managing multiple Kubernetes clusters. 
+
+If you plan to deploy and operate a large deployment with multiple
+Kubernetes clusters, you can use the {{ product }}'s CAPI providers for AWS and
+[MAAS]. Follow the [cluster-provisioning-with-CAPI guide] to deploy and operate
```

Comment:
I'd avoid naming the specific infrastructure providers here, technically other infrastructure providers could be used beyond what we have tested and put in as examples. This also feels it can get outdated quick with new infra providers tested/added to docs.